src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.DocAsCode.Build.ManagedReference
     [Export(nameof(ManagedReferenceDocumentProcessor), typeof(IDocumentBuildStep))]
     public class FillReferenceInformation : BaseDocumentBuildStep, ICanTraceContextInfoBuildStep
     {
+        private IEnumerable<SourceInfo> _lastContextInfo;
         private Dictionary<string, SourceInfo> _items = new Dictionary<string, SourceInfo>();
```

Comment:
Only store what the buildstep needs?

---
In reply to: [107822615](https://github.com/dotnet/docfx/pull/1466#discussion_r107822615) [](ancestors = 107822615)